app/grandchallenge/reader_studies/models.py

```diff
@@ -339,39 +343,47 @@ def get_progress_for_user(self, user):
 
         expected = hanging_list_count * self.answerable_question_count
         answers = Answer.objects.filter(
-            question__in=self.answerable_questions, creator_id=user.id
+            question__in=self.answerable_questions,
+            creator_id=user.id,
+            is_ground_truth=False,
         ).distinct()
         answer_count = answers.count()
 
-        # There are unanswered questions
-        if answer_count % self.answerable_question_count != 0:
-            # Group the answers by images and filter out the images that
-            # have an inadequate amount of answers
-            unanswered_images = (
-                answers.order_by("images__name")
-                .values("images__name")
-                .annotate(answer_count=Count("images__name"))
-                .filter(answer_count__lt=self.answerable_question_count)
-            )
-            image_names = set(
-                unanswered_images.values_list("images__name", flat=True)
-            ).union(
-                set(
-                    Image.objects.filter(
-                        readerstudies=self, answers__isnull=True
+        # Group the answers by images and filter out the images that
+        # have an inadequate amount of answers
+        unanswered_images = (
+            answers.order_by("images__name")
+            .values("images__name")
+            .annotate(answer_count=Count("images__name"))
+            .filter(answer_count__lt=self.answerable_question_count)
+        )
+        image_names = set(
```

Comment:
Wow! Nothing to change here but I think that we probably need a model manager at this point.

Reply:
Added an issue for this: https://github.com/comic/grand-challenge.org/issues/1154